### ISSUE TITLE
Document support for non-stdlib function/objects

### ISSUE THREAD

**Issue by ashi009** [grafana/jsonnet-language-server#109]
Currently only stdlib has doc support, it would be nice to have the same feature for user defined libraries.

**Comment by nogweii** [comment]
Would this be through understanding [docsonnet](https://github.com/jsonnet-libs/docsonnet/) annotations?

**Comment by jemag** [comment]
I think supporting docsonnet would be a very significant improvement to UX. Currently, while I may check the function definition through `hover` I always need to search within the library code itself to obtain the documentation. 

Being able to obtain that documentation directly with `hover` as in other languages would be amazing.

**Comment by tmeijn** [comment]
Indeed. Just starting with jsonnet and the LSP and this would help tremendously with navigating the Grafana library and would encourage to start documenting your own libraries with docsonnet.
